lib/shelf_pubserver.dart

```diff
@@ -471,37 +460,27 @@ class ShelfPubServer {
         headers: {'content-type': 'application/json'});
   }
 
-  Future<shelf.Response> _unauthorizedRequest() async {
-    return new shelf.Response(403,
-        body: JSON.encode({
-          'error': {'message': 'Unauthorized request.'}
-        }),
-        headers: {'content-type': 'application/json'});
-  }
+  shelf.Response _unauthorizedRequest() => new shelf.Response(403,
```

Comment:
If it doesn't fit on one line you should consider using a block - which makes these more readable - just as before.

Reply:
I disagree. Even wrapped, I find `=>` more readable even when wrapped.